packages/ai-core/src/browser/ai-configuration/agent-configuration-widget.tsx

```diff
@@ -83,6 +85,12 @@ export class AIAgentConfigurationWidget extends ReactWidget {
         </div>;
     }
 
+    private renderAgentName(agent: Agent): string {
+        // Check if the agent has tags, and join them with a comma separator if they exist
+        const tagsSuffix = agent.tags?.length ? ` (${agent.tags.join(', ')})` : '';
+        return `${agent.name}${tagsSuffix}`;
+    }
```

Comment:
We may want to render those tags more nicely, but this is optional.

```suggestion
    private renderAgentName(agent: Agent): React.ReactNode {
        const tagsSuffix = agent.tags?.length ? <span>{agent.tags.map(tag => <span className='agent-tag'>{tag}</span>)}</span> : '';
        return <span>{agent.name} {tagsSuffix}</span>;
    }
```

![image](https://github.com/user-attachments/assets/eb66ab3f-87a9-400f-b2d9-5587efa563a8)

To make them look as above, we need the following CSS in `packages/ai-core/src/browser/style/index.css`:
```css
  padding: calc(var(--theia-ui-padding) * 2 / 3);
  padding-top: 0px;
  padding-bottom: 0px;
  border-radius: calc(var(--theia-ui-padding) * 2 / 3);
  background: hsla(0, 0%, 68%, 0.31);
```

Reply:
looks very nice, thank you! applied in new commit